### PR TITLE
Add prismic ads

### PR DESCRIPTION
**Pull request by jaredcbaum**
Includes prismic-dom and prismic-javascript dependencies.
Adds prismic ads on to right sidebar.
Ads wont display if the right sidebar is too tall already.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -38,13 +38,16 @@
...
     "natives": "^1.1.6",
     "node-fetch": "^2.6.0",
-    "prismic-javascript": "^2.7.1",
     "prismjs": "^1.16.0",
     "rollup-plugin-dotenv": "^0.3.0",
     "stencil-router-v2": "^0.5.0",
     "typedoc": "^0.17.7",
     "workbox-build": "4.3.1"
   },
+  "dependencies": {
```
Comment (perrygovier) [inline package.json:47]: Can these be under dev-dependencies? Technically, there's a lot that should be moved to "dependencies", but it doesn't make sense to have these two be the only ones.

File: src/components/in-page-navigation/in-page-navigation.tsx [hunk 2]
```diff
@@ -24,14 +31,26 @@ export class InPageNavigtion implements ComponentInterface {
...
+      this.adEl.style.height = 'auto';
+    } else {
+      this.adEl.style.height = '0px';
+    }
+  }
+  
 
   @Watch('pageLinks')
-  @Listen('resize', { target: 'window' })
+  @Listen('resize', { target: 'window'})
```
Comment (perrygovier) [inline src/components/in-page-navigation/in-page-navigation.tsx:53]: Let's get ya set up with the same linter the rest of us are using

File: package.json (hunk 1 above)
Comment (jaredcbaum) [reply package.json:47]: I thought dependencies are the ones that are required at runtime?
https://github.com/ionic-team/capacitor-site/blob/add-prismic-ads/src/components/internal-ad/internal-ad.tsx#L58

Comment (jaredcbaum) [reply package.json]: Well looks to still work when under dev dependencies

**Comment by mlynch** [comment]
🚀

**Comment by mlynch** [comment]
I took out `PrismicDOM` because we have `PrismicRichText` which can do that more safely and compatible w/ JSX/vdom:

```typescript
<PrismicRichText richText={this.ad.ad_copy} /
```

**Comment by mlynch** [comment]
What is the purpose of the `internalAdLoaded` event and the `componentDidRender` handler in `in-page-navigation.tsx`? Seems like there's probably a better way to achieve what this is doing that is more idiomatic and without needing to do any DOM manipulation

**Comment by jaredcbaum** [comment]
It's because once the ad component loads, I'm checking again if it causes the <in-page-navigation> to overflow.  If it does, then the ad will have a height of 0.  Every time the ad is loaded it checks again to see if there's space.  The logic was a little roundabout because the <internal-ad> has to be loaded in order to get the height, and then only it it fits the height is changed to 'auto'.

**Pull request merged**